### ISSUE TITLE
[Go] Initial DX should always be first entry in timeline

### ISSUE THREAD

**Issue by regrounding** [jliphard/CancerBase#54]
Both my initial diagnosis and my first treatment ocurred in 2/2002. They are not showing in the correct order...

![screen shot 2016-07-12 at 5 36 29 pm](https://cloud.githubusercontent.com/assets/20259990/16788169/8aa260fe-4857-11e6-9b05-be46be15a689.png)

**Comment by jliphard** [comment]
This is a very difficult problem to get right in general. Since we are only asking for month level data (since most people cannot remember the precise date) there will in general be situations were multiple diagnoses, treatments, and events take place in the same month. This does not matter from a scientific perspective, but matters to the users since they want things to be in the "right" sequential order. 

The fast and easy solution in this case is to start all timelines with the initial DX, which presumably always precedes treatment etc. The general solution, which could be implemented at some point, is to allow users to manually move entries up and down in the timeline to resolve situations in which multiple things started in the same months. 

I would recommend a the simple fix first (Initial DX should always be first entry in timeline) and then delay the "general" solution - manual dragging/movement of events until things are less crazy.

**Comment by regrounding** [comment]
I agree, Jan. We may also need a disclaimer somewhere that explains that the entire site is a co-collaboration and a work in progress. Drag and drop down the road will be a beautiful thing! 

FWIW, this one happens to be mine, but each time I interact with someone and explain "we will keep it on our list" I get a very warm response.
